mysql-test/t/percona_bug_ps6113.test

```diff
@@ -0,0 +1,29 @@
+#
+# Bug #97828
+# PS-6113: If ANALYZE TABLE runs more than 600 seconds diagnostic query may crash server
+#
+
+--source include/have_innodb.inc
+
+CREATE TABLE t1(a int, index inda(a)) ENGINE=INNODB;
+INSERT INTO t1 VALUES(1);
+
+SET DEBUG_SYNC='innodb_dict0stats_dict_stats_update_persistent SIGNAL analyze.running WAIT_FOR analyze.finish';
+--send ANALYZE TABLE t1
+
+# ANALYZE TABLE is running. Query for stats.
+
+--connect(con1, localhost, root)
+SET DEBUG_SYNC='now WAIT_FOR analyze.running';
+
+--replace_column 1 ENGINE 2 D_I_LENGTH 3 ENGINE 4 DATA_LENGTH 5 INDEX_LENGTH
```

Comment:
do we need to mask them ? Aren't they constant?

Reply:
done